tests/acl/test_stress_acl.py

```diff
@@ -35,6 +36,34 @@
 ACL_RULE_NUMS = 10
 
 
+@pytest.fixture(scope="module", autouse=True)
+def remove_dataacl_table(duthosts):
+    """
+    Remove DATAACL to free TCAM resources.
+    The change is written to configdb as we don't want DATAACL recovered after reboot
+    """
+    TABLE_NAME = "DATAACL"
+    for duthost in duthosts:
+        lines = duthost.shell(cmd="show acl table {}".format(TABLE_NAME))['stdout_lines']
+        data_acl_existing = False
+        for line in lines:
+            if TABLE_NAME in line:
+                data_acl_existing = True
+                break
+        if data_acl_existing:
+            # Remove DATAACL
+            logger.info("Removing ACL table {}".format(TABLE_NAME))
+            cmds = [
+                "config acl remove table {}".format(TABLE_NAME),
+                "config save -y"
+            ]
+            duthost.shell_cmds(cmds=cmds)
+    yield
+    # Recover DUT by reloading minigraph
+    for duthost in duthosts:
+        config_reload(duthost, config_source="minigraph")
```

Comment:
1. Config reload takes time. Is there a faster way to recover DATAACL after testing?
2. If config reload is the only way to recover configuration, please use config source from running_golden_config:
```
config_reload(duthost, config_source="running_golden_config"
```
The reason is that there are some pretests updated a few configurations and saved the running config to "running_golden_config". We have a common fixture always compare the running config with the running golden config to detect any configuration not recovered by the current script to avoid affecting subsequent test scripts. If there are any configuration differences, the common fixture will do a reload.
So, if a test does not restore the configuration to running config, the common fixture has to do a config reload again. This will unnecessarily increase test time.

Reply:
Thanks for your comments, i have removed config reload and used other method to reconfigure the acl.